templates/error.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Error</title>
```

Comment:
Recommend a filename of "upload_error.html" for this template, since it's specific to CSV uploads.  And make the title specific, too, such as "CSV Upload Error".